### PR TITLE
ENG-827 finalze pipelines

### PR DESCRIPTION
**Pull request by ampie**
The PR illustrates how we would be using the new fine grained release artifacts from the new fine grained git repos. It is still a bit rough around the edges as we need to implement some exclusions in the source poms. But this version seems to work.
Please check this out, run `mvn clean package jetty:run-war -Pjetty-local -Pderby` to test for local development.
Also play around with the various Docker build options

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pom.xml [hunk 1]
```diff
@@ -43,14 +31,25 @@
...
-        </maven.snapshot.repository.url>
-        <maven.release.repository.url>https://oss.sonatype.org/service/local/staging/deploy/maven2/
-        </maven.release.repository.url>
+        <entando-engine.version>6.2.9</entando-engine.version>
+        <entando-admin-console.version>6.2.10</entando-admin-console.version>
+        <entando-portal-ui.version>6.2.4</entando-portal-ui.version>
+        <entando-plugin-jacms.version>6.2.8</entando-plugin-jacms.version>
+        <entando-plugin-jpversioning.version>6.2.3</entando-plugin-jpversioning.version>
+        <entando-plugin-jpseo.version>6.2.2</entando-plugin-jpseo.version>
+        <entando.keycloak.plugin.version>6.2.0-SNAPSHOT</entando.keycloak.plugin.version>
```
Comment (Kerruba) [inline pom.xml:52]: Is this correct?

File: pom.xml [hunk 2]
```diff
@@ -548,7 +642,7 @@
                             <name>entando/${project.artifactId}-${jee.server}:${project.version}</name>
                             <alias>${project.artifactId}-engine</alias>
                             <build>
-                                <from>entando/${server.base.image}:6.1.5</from>
+                                <from>entando/${server.base.image}:6.1.3</from>
```
Comment (Kerruba) [inline pom.xml]: We downgraded the version?

**Review by Kerruba — COMMENTED** [review]
Some questions

File: pom.xml (hunk 2 above)
Comment (ampie) [reply pom.xml]: Thanks well spotted

File: pom.xml (hunk 1 above)
Comment (ampie) [reply pom.xml:52]: Yes. Assuming your are referring to the keycloak SNAPSHOT  that pipeline still needs to be done

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-de-app/pull/27#issuecomment-655956445" title="Approved">Kerruba</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)~~ [Kerruba]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**